From c0d727c16323c836808d31824acdafa650e95c77 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Tue, 2 Oct 2018 14:34:07 +0200
Subject: [PATCH 0704/1051] arm64: dts: marvell: armada-7040/8040-db: fix
 labels

The mainline labels of the nand and serial nodes are
different, so fix imported device trees for alternative
variants of Armada 7040/8040 DB.

Change-Id: I89c28273648bf4d3648123f0e06d0f68ffc2d0c1
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60007
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-D.dts | 2 +-
 arch/arm64/boot/dts/marvell/armada-8040-db-D.dts | 2 +-
 arch/arm64/boot/dts/marvell/armada-8040-db-E.dts | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
index 4243053b1392..00c13b572e7b 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
@@ -68,7 +68,7 @@
 	};
 };
 
-&cp0_nand {
+&cp0_nand_controller {
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
index 0e0871dccead..cee09bd7b36d 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
@@ -106,7 +106,7 @@
 };
 
 
-&cp1_nand {
+&cp1_nand_controller {
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
index a3199682d936..2f8f01079c86 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
@@ -97,11 +97,11 @@
 	};
 };
 
-&cp0_serial0 {
+&cp0_uart0 {
 	status = "okay";
 };
 
-&cp0_serial1 {
+&cp0_uart1 {
 	status = "okay";
 };
 
-- 
2.17.1

